resources/rust/main.rs

```diff
@@ -0,0 +1,17 @@
+// Import the interface to Tessel hardware
+extern crate rust_tessel as tessel;
+
+fn main() {
+  
+  // Set the led pins as outputs with initial states
+  let led1 = tessel.led[0].output(1);
```

Comment:
Thanks for the suggestion, @kevinmehall.

It looks like it won't be quite so easy because I don't think [the `tessel` lib](https://github.com/tessel/tessel-rust) was implemented in such a way that you could call `new` on it. It probably needs updating to newer versions of Rust as well.